src/main/java/org/jabref/gui/SendAsEMailAction.java

```diff
@@ -107,19 +111,30 @@ private String sendEmail() throws Exception {
             }
         }
 
-        String mailTo = "?Body=".concat(rawEntries.toString());
-        mailTo = mailTo.concat("&Subject=");
-        mailTo = mailTo.concat(preferencesService.getExternalApplicationsPreferences().getEmailSubject());
-        for (String path : attachments) {
-            mailTo = mailTo.concat("&Attachment=\"").concat(path);
-            mailTo = mailTo.concat("\"");
-        }
-
-        URI uriMailTo = new URI("mailto", mailTo, null);
+        URI uriMailTo = getUriMailTo(rawEntries, attachments);
 
         Desktop desktop = Desktop.getDesktop();
         desktop.mail(uriMailTo);
 
         return String.format("%s: %d", Localization.lang("Entries added to an email"), entries.size());
     }
+
+    private URI getUriMailTo(StringWriter rawEntries, List<String> attachments) throws URISyntaxException {
+        StringBuilder mailTo = new StringBuilder();
+        if (action == StandardActions.SEND_TO_KINDLE) {
+            mailTo.append(preferencesService.getExternalApplicationsPreferences().getKindleEmail());
```

Comment:
I assume this also works if the email is not configured?

Reply:
Currently it does work without a configured email and opens the email dialog with a blank mailTo address. Should this behavior be changed?

Reply:
Hm, I think it's oaky. one can still add the mail in the email program